### PR TITLE
AWS Cloudwatch metrics role

### PR DESCRIPTION
**Pull request by bonclay7**
Tasks list:
- [x] Base source for module
- [x] Tests
- [x] Documentation
- [x] Support for many os families (disclaimer added in readme)

### CODE REVIEW

**Comment by ajacoutot** [comment]
I am not sure we want to use a "roles" sub-directory. We should just include the roles themselves under the top dir.
Also we need to agree on a proper naming so that we easily know what's a role does.
e.g. aws-cloudwatch-metrics or so

**Comment by ajacoutot** [comment]
And @sylicia should comment as well ;-)

**Comment by bonclay7** [comment]
Thanks for the comments, I'm not an ansible pro, so much appreciated !

**Comment by sylicia** [comment]
@bonclay7 thank's for your readmes. :)
I like this naming convention and the way to do some tests.

Concerning variables, it would be better to set them in the test playbook because if they are set them in the role, they cannot be overwritten.

It is interesting to have default values.
If you want default values, you can set your variables in "defaults" directory instead of "vars".

**Comment by bonclay7** [comment]
@ajacoutot @sylicia ready to merge if you're ok

File: aws-cloudwatch-metrics/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+---
```
Comment (sylicia) [inline aws-cloudwatch-metrics/vars/main.yml]: Could you remove this file if it is unused?

**Review by ajacoutot — APPROVED** [review]
@bonclay7 just merge this; we can tweak whatever needs tweaking later. Thanks :-)

**Pull request merged**